Documentation/goldsrc-protocol-support.md

```diff
@@ -6,9 +6,10 @@ For connecting to GoldSrc-based servers, use this command:
 connect ip:port gs
 ```
 
-But keep in mind, there are requirement for server to be able to accept connections from Xash3D-based clients: it should accept HLTV connections.
+But keep in mind, there are requirement for server to be able to accept connections from Xash3D-based clients: it should use Reunion or Dproto.
```

Comment:
Dproto by default enables fake client detection. I tried to fix it, it just drops player after a while.

Reunion is fine because it doesn't do anything against fake clients, instead it's done in ReAuthCheck, which is okay-ish, though sometimes it also kicks the player anyway.

Btw since we're here, maybe add GoldSrc protocol support to the README.md?

Reply:
> Btw since we're here, maybe add GoldSrc protocol support to the README.md?

Maybe I misundertood, but it's already there.